src/js/components/Menu.js

```diff
@@ -176,6 +176,11 @@ class MenuDrop extends Component {
 
     delete boxProps.onClick;
 
+    if (this.props.size) {
```

Comment:
I'm wondering if we need the if clause. For me `delete boxProps.size` would suffice.